code/index.html

```diff
@@ -1,22 +1,93 @@
 <!DOCTYPE html>
 <html>
+
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <title>Travel insights</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <meta name="viewport" content= "width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/6f68b8218c.js" crossorigin="anonymous"></script>
   </head>
+
   <body>
     <header>
-      <!-- Some kind of logo and navbar -->
+      <img src="suitcase-logo1.jpeg" alt="Airplane logo">
+      <div class="site-name">TRAVEL <br> INSIGHTS
+      </div>
+       <nav>
+         <a href="#air">AIR</a>
+         <a href="#hotel">HOTEL</a>
+         <a href="#car">CAR</a>
+         <a href="#leisure">LEISURE</a>
+       </nav>
+       <div class="hamburger">
+        <i class="fas fa-bars fa-2x"></i>
+       </div>
     </header>
-
```

Comment:
I learned today that it could be a good idea to insert `<main>` after the header and before the footer